pkg/security/tests/schemas.go

```diff
@@ -107,6 +107,10 @@ func (tm *testModule) validateExitSchema(t *testing.T, event *model.Event) bool
 
 //nolint:deadcode,unused
 func (tm *testModule) validateOpenSchema(t *testing.T, event *model.Event) bool {
+	if ebpfLessEnabled {
+		return true
+	}
```

Comment:
should make the same update on other schema validation ? at least for those where there is now a "if !ebpflessenabled" to skip them.